### PR TITLE
PT-3127: Rearrange pedigree import dialog to put the options above the actual file selection

### PR DESCRIPTION
**Pull request by AlexLapena**
This also redesigns the dialogue so that the file selection is similar to patient JSON import.

### CODE REVIEW

**Comment by AlexLapena** [comment]
@sashaandjic brought up an interesting point that our JSON import menu looks a lot better than our current pedigree import. I think rather than fixing the current spacing, we re-order the bottom section to say `Paste a {format} in the box below, or [Select A Local File]`

**Comment by AlexLapena** [comment]
Furthermore, we also discussed that in the future, we could implement some sort of drop down menu for `Data Format` and `Options`. Currently, the import menu looks a bit less friendly than an import menu should, in my opinion.

**Comment by sashaandjic** [comment]
It would also make sense to do the IMPORT and CANCEL buttons alignments the same way as in the Import dialog.

**Comment by sashaandjic** [comment]
Do we need "," between "below" and SELECT LOCAL FILE bttn?

**Comment by AlexLapena** [comment]
Fixed UI.

**Comment by sashaandjic** [comment]
looks great. 2 minor things:
- remove comma from "Paste data in the box below, or"
- move the "Paste dta..." and the corresponding text box few lines down

**Comment by AlexLapena** [comment]
@sashaandjic Fixed.

File: components/pedigree/resources/src/main/resources/pedigree/editor.css [hunk 1]
```diff
@@ -460,12 +459,14 @@
   margin-top: 4px !important;
 }
 .import-file-button-wrapper {
-  margin-left: -494px;
+  position: relative;
+  margin-left: -350px;
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/editor.css]: This is extremely fragile... This whole CSS must be rewritten.

File: components/pedigree/resources/src/main/resources/pedigree/editor.css [hunk 2]
```diff
@@ -460,12 +459,14 @@
...
 }
 .import-block-bottom {
   margin: 1.2em 0 0.5em 0;
 }
 .import-section {
+  margin-top: 0.5em;
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/editor.css]: I don't like this, now the label and the radio buttons are misaligned.

**Comment by sdumitriu** [comment]
Misaligned label:

screenshot_20170804_001714

**Comment by AlexLapena** [comment]
I wasn't able to have enough time tonight to address all the issues unfortunately, but I will rewrite the fragile CSS part asap.
label

**Pull request merged**